src/applications/edu-benefits/0994/containers/IntroductionPage.jsx

```diff
@@ -5,51 +5,38 @@ import FormTitle from 'platform/forms-system/src/js/components/FormTitle';
 import SaveInProgressIntro from 'platform/forms/save-in-progress/SaveInProgressIntro';
 import CallToActionWidget from 'platform/site-wide/cta-widget';
 import WizardContainer from '../../wizard/containers/WizardContainer';
-import {
-  WIZARD_STATUS_NOT_STARTED,
-  WIZARD_STATUS_COMPLETE,
-} from 'applications/static-pages/wizard';
+import { WIZARD_STATUS_COMPLETE } from 'applications/static-pages/wizard';
 import { connect } from 'react-redux';
 import { showEduBenefits0994Wizard } from '../../selectors/educationWizard';
 
 export class IntroductionPage extends React.Component {
-  state = {
-    wizardStatus:
-      sessionStorage.getItem('wizardStatus') || WIZARD_STATUS_NOT_STARTED,
-  };
+  state = { status: sessionStorage.getItem('wizardStatus') };
 
   componentDidMount() {
     focusElement('.va-nav-breadcrumbs-list');
   }
 
   setWizardStatus = value => {
     sessionStorage.setItem('wizardStatus', value);
-    this.setState({ wizardStatus: value });
+    this.setState({ status: value });
   };
 
   render() {
-    const resBurden = '10';
-    const ombNumber = '2900-0866';
-    const expDate = '04/30/2022';
-    const { wizardStatus } = this.state;
-    const { shouldEduBenefits0994WizardShow } = this.props;
-    const shouldSubwayMapShow =
-      !shouldEduBenefits0994WizardShow ||
-      wizardStatus === WIZARD_STATUS_COMPLETE;
-    const shouldWizardShow =
-      shouldEduBenefits0994WizardShow &&
-      wizardStatus !== WIZARD_STATUS_COMPLETE;
+    const { status } = this.state;
+    const { showWizard } = this.props;
+    const show = showWizard && status !== WIZARD_STATUS_COMPLETE;
+
+    if (showWizard === undefined) return null;
```

Comment:
Shouldn't this allow the subway map to render? Or is this only `undefined` before the component has mounted?

Reply:
yep because were toggling the state of the view we need to render `null` here until the component mounts